history/characters/1_azerothian.txt

```diff
@@ -17177,4 +17178,110 @@
 		add_trait=lunatic
 	}
 	608.3.10={ death=yes }
+}
+
+#dynasty=413
+5816={
+	name=Isiah			#Reginald's greatgrandfather
+	dna="0cebc000ca0"
+	properties="ij0jgh0000000000000000"
+	properties="we0jga0000000000000000"
+	culture=azerothian religion=holy_light
+	martial=4 diplomacy=6 stewardship=3 intrigue=4 learning=4
+	495.10.8 = { birth=yes add_trait=creature_human }
+	522.7.13 = {
+		add_spouse=5821		#Reginald's grandmother
+	}
+	550.6.13={ death=yes }
+}
+5817={
+	name=Landen			#Reginald's grandfather
+	dynasty=413
+	dna="0lahj0f0c00"
+	properties="we0jga0000000000000000"
+	culture=azerothian religion=holy_light
+	martial=4 diplomacy=6 stewardship=3 intrigue=4 learning=4
+	father=5816		# Isiah
+	mother=5820		# Damia
+	523.10.8 = { birth=yes add_trait=creature_human }
+	546.7.13 = {
+		add_spouse=5821		#Reginald's grandmother
```

Comment:
Typo. Both characters marry the same wife.

Reply:
fixed